### PR TITLE
Feature/new frontpage design

### PR DESCRIPTION
**Pull request by Bluesmile82**
Back to make some tweaks to the landing page design

- [x] Added Global Forest Watch navbar and footer via script tags and empty divs
- [x] Added Social Footer component
- [x] Fixed some styles and move button

---
In localhost the language dropdown but in staging or production it should show the languages with google translations.

Social footer component has slight differents in design. It has its own component and stylesheet.

Missing fix responsive design or link directly to download app: https://www.npmjs.com/package/mobile-detect

image

### CODE REVIEW

File: src/components/pages/landing/Landing.js [hunk 1]
```diff
@@ -1,44 +1,59 @@
...
 import { FormattedMessage } from 'react-intl';
 import { Link } from 'react-router-dom';
+import SocialFooter from './SocialFooter';
 
 class Landing extends React.Component {
+  componentDidMount(){
+    this.script = document.createElement('script');
+    this.script.type = 'text/javascript';
+    this.script.async = true;
+    this.script.innerHTML = "window.liveSettings = {picker: '#transifexTranslateElement',api_key: '9eda410a7db74687ba40771c56abd357',detectlang: false,site: 'gfw-watcher'};";
```
Comment (sorodrigo) [inline src/components/pages/landing/Landing.js]: Please move the api_key and use a env var instead, also I would save each script and config options in a constant, to separate it from the component.

File: src/components/pages/landing/SocialFooter.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+class SocialFooter extends React.Component {
+  componentDidMount(){
+    this.script = document.createElement('script');
+    this.script.type = 'text/javascript';
+    this.script.async = true;
+    this.script.innerHTML = `
```
Comment (sorodrigo) [inline src/components/pages/landing/SocialFooter.js]: Same here, please move this to a constant.

File: src/components/pages/landing/SocialFooter.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          </div>
+          <div className="small-6 columns">
+            <div className="text -title-xxs -color-5">Sign up to receive updates</div>
+            <div className="c-social__icons">
+              <a href="?show_newsletter=true" title="Receive Global Forest Watch updates" onclick="ga('send', 'event', 'Footer', 'Click', 'Subscribe');">
+                <svg className="icon">
+                  <use xlinkHref="#icon-mail" />
+                </svg>
+              </a>
+              <a href="https://twitter.com/globalforests" title="Follow @globalforests" target="_blank" onclick="ga('send', 'event', 'Footer', 'Click', 'Twitter');">
```
Comment (sorodrigo) [inline src/components/pages/landing/SocialFooter.js]: This is giving me an error on the console, the correct property is `onClick`.
When I click it I can't see the languages, do I need something else on my env file?

File: src/styles/components/_c-landing.scss [hunk 4]
```diff
@@ -19,41 +26,39 @@
       bottom: 0;
     }
     .phone-image {
```
Comment (sorodrigo) [inline src/styles/components/_c-landing.scss]: About this styles, maybe a solution for the resizing issue can be to set the screenshot image as a child of the phone image and make the position of it relative to its parent.

File: src/styles/components/_c-landing.scss [hunk 5]
```diff
@@ -19,41 +26,39 @@
...
-      left: 0;
-      height: 788px;
-      background-size: contain;
       background-image: url(/images/phone@2x.png) !important;
```
Comment (sorodrigo) [inline src/styles/components/_c-landing.scss]: Unless you're dealing with styles set by a third-party library and there's no other resort, avoid using `!important`. In this case I think you can remove it and nothing will change.

**Review by sorodrigo — CHANGES_REQUESTED** [review]
Apart from the comments on the code, I found the following issue:
If I click on _Access Desktop App_ and then I navigate back the navigation bar wont load anymore.

File: src/components/pages/landing/SocialFooter.js (hunk 3 above)
Comment (Bluesmile82) [reply src/components/pages/landing/SocialFooter.js]: If you are talking about the language dropdown. Hector said that it wouldn't work on local

**Pull request merged**